### ISSUE TITLE
Bot not responding with default MSG to text that that is not recognized.

### ISSUE THREAD

**Issue by bkurowski** [samtecspg/articulate#413]
### Issues

Hi ! I ran into small problem, for which I can not find a solution.
When I send a message for example "No" which is not trained inside Articulate at all.
I get no answer at all. Visible Below.

![sample_message](https://user-images.githubusercontent.com/40358189/49079657-1d187580-f239-11e8-8022-12749e4316e3.PNG)

If I send a message that is already trained for example "Hello" or any other query I get answer no problem.
Shouldn't the Articulate come back to me "Sorry can you rephrase that" ? If it doesn't understand something ? 
I receive an error like this. But it doesn't tell me much.

![error](https://user-images.githubusercontent.com/40358189/49079739-5650e580-f239-11e8-9292-6b557fce8649.PNG)

Thanks ! 

- Did you download a release or pull from Master? Which release?
--> v0.12.3
- What operating system are you running?
--> Ubuntu
- Are you running locally or in the cloud?
--> Locally
- Which browser are you using?
--> Chrome

----

**Comment by wrathagom** [comment]
@bkurowski are you manipulating the context via the API at all? Are you sending `No` to the `/converse` endpoint or are you using the direct input syntax `\intentName@0.6{"entity": "value"}`? Can I see more of the conversation?

Slots being undefined is an error I can't remember seeing before and don't know if I will be able to er-produce without more info.

**Comment by bkurowski** [comment]
I am using `/converse` API.
I have observed that when I send message to "Articulate chat window"  I receive response saying"I am still learning" but using custom chat does not allow that response to come back.
It is weird action, as it responds for other messages. 
This is some bigger conversation. 

![chat](https://user-images.githubusercontent.com/40358189/49081031-61a61000-f23d-11e8-8869-c667438afedf.PNG)
 
![capture1](https://user-images.githubusercontent.com/40358189/49081044-6f5b9580-f23d-11e8-8271-492e7a1233b3.PNG)

Let me know if you need any other info.

Thanks.

**Comment by wrathagom** [comment]
Are you including slots in your response payload? It won't always exist, so you need to add handlebars to handle the case when it doesn't.

**Comment by bkurowski** [comment]
I do not add any slots nor need handlebars in it. What I mean is that I should receive an answer of "Sorry can you rephrase that" and I expect that to be received because I do not have any intents or entities for word "No". 
I only need to find out why, when sending that message "No" to API /converse, Articulate doesn't come back with the default message "Sorry can you rephrase that". I am receiving status error 500. 
Where as I am sending other message that is already trained, I do not encounter that problem. 

Trying to understand where the problem might be. 
I am able to handle that error inside my flow, but Articulate should reply automatically if it understands my query or doesn't.
Am I correct  ?

**Comment by wrathagom** [comment]
Do you use a custom response payload, that is what I am asking.

![image](https://user-images.githubusercontent.com/1779894/49181112-81643380-f324-11e8-9612-ceaa6c92cccc.png)

**Comment by bkurowski** [comment]
No, I am not using custom response payload. 
Sorry for late answer I was off.

**Comment by wrathagom** [comment]
@bkurowski did we ever crack this one?

**Comment by bkurowski** [comment]
To be honest it disappeared. Does not happen anymore. Not sure what i have done. 
But cheers anyway ! 

I'd say it can be closed. If it happens again I will be doing research then.
